chain-tx-enclave/tx-query/enclave/src/attest.rs

```diff
@@ -601,16 +601,31 @@ fn renew_ra_cert(global_ra_cert: &mut RACache) -> Result<(), RAError> {
         .create_key_pair()
         .map_err(|_| RAError::CertGeneration)?;
 
-    let (attn_report, sig, cert) = match create_attestation_report(
-        &pub_k,
-        &ias_key,
-        sgx_quote_sign_type_t::SGX_UNLINKABLE_SIGNATURE,
-    ) {
-        Ok(r) => r,
-        Err(_e) => {
-            #[cfg(not(feature = "production"))]
-            println!("Error in create_attestation_report: {:?}", _e);
-            return Err(RAError::CertGeneration);
+    // https://software.intel.com/en-us/node/802449
+    // when there are some network connection error, SGX_ERROR_BUSY will return
+    // we should wait (typically several seconds to tens of seconds) and retry again
+    let mut retry = 3;
+    let (attn_report, sig, cert) = loop {
```

Comment:
Should we add `sleep` between retries?

Reply:
It has this busy-wait: https://github.com/crypto-com/chain/pull/836/files#diff-8d3779693d563d7fa015558c7dad8e36R620 

@linfeng-crypto it may be cleaner if you add `thread` feature in tstd dependency and just use sleep: https://github.com/apache/incubator-teaclave-sgx-sdk/blob/master/sgx_tstd/src/thread/mod.rs#L142